.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "dleq/cgo-dleq"]
 	path = dleq/cgo-dleq
-	url = https://github.com/noot/cgo-dleq.git
+	url = https://github.com/athanorlabs/cgo-dleq.git
```

Comment:
There is a second level submodule that would need to be updated in the `cgo-dleq` repo:
```
./dleq/cgo-dleq/lib/c-dleq/.git
```
In that repo, you can run this to update it:
```
cgo-dleq$ git submodule set-url -- lib/c-dleq 'https://github.com/athanorlabs/c-dleq.git'
```

Reply:
updated: https://github.com/AthanorLabs/cgo-dleq/commit/e9086d079c16a715dedba30a44460208a42057ec